src/app/shop/[category]/[productId]/_components/TabContents/DetailTab.tsx

```diff
@@ -19,7 +19,7 @@ interface TabData {
 interface DetailTabProps {
   detailsImg: string;
   reviewData: ProductReviewType;
-  productId: string;
+  productId: number;
```

Comment:
productId.toString() 메서드로 받으시는경우 있는거같은데 
```suggestion
  productId: number | string;
```
그냥 함수에서 둘다받아도 상관없을거같아요